### PR TITLE
Add some numbers for RFC and remove some references to make space

### PR DESCRIPTION
**Pull request by timj**
and some other cleanups.

### CODE REVIEW

File: rfc.tex [hunk 1]
```diff
@@ -25,11 +25,12 @@ \section{Decision Making Process}
 The DM Systems Engineer is tasked with looking at RFCs on a weekly basis to ensure that proposed RFCs are not languishing and that implemented RFCs are correctly marked as such.
 The \emph{Implemented} state is recognized by scanning all the \emph{Adopted} RFCs and seeing that all triggered work has been completed.
 
-The RFC process has been extremely successful with N RFCs filed since 201? and M implemented.
-We have found that some RFCs are filed asking for changes that the proposer feels are a good change to make but which they themselves are not going to be repsonsible for implementing.
+The RFC process has been extremely successful with 467 RFCs filed since 2014, with 321 implemented and 44 withdrawn.
```
Comment (ktlim) [inline rfc.tex]: Begs the question of what is going on with the other 102...

File: rfc.tex [hunk 2]
```diff
@@ -11,7 +11,7 @@ \section{Decision Making Process}
 
 [show Jira RFC workflow here?]
 
-In 20?? we decided to change our approach to the latter process.
+In 2014 we decided to change our approach to the latter process.
 DM developers have to be familiar with Jira and we developed a Jira workflow for ``Request for Comments'' (RFCs).
 RFCs were designed to empower individual developers to propose decisions.
 Emphasis is placed on the developer being responsible for any fall out from a change.
```
Comment (ktlim) [inline rfc.tex:15]: This is a bit misleading, and the next sentence is false.  RFCs can be used to propose work for others; the main point was supposed to be that if you're going to do all the work yourself, you don't even need an RFC (unless it affects others).

File: rfc.tex (hunk 1 above)
Comment (timj) [reply rfc.tex]: 89 adopted, 5 flagged. 7 retired (which is not quite the same as Withdrawn but may as well be) 1 is proposed.

File: rfc.tex [hunk 3]
```diff
@@ -40,5 +41,5 @@ \subsection{Change Control Board}
 Inside DM we have our own formal CCB that has evolved over the years from the SAT to something we called the Technical Control Team (TCT) but which has now been reorganized as the DM CCB.
 The DM CCB is chaired by the DM Systems Engineer and has representation from senior members of each DM team as well as the DM Project Manager and DM Subsystem Scientist.
 DM CCB uses the same RFC process as described above, but issues to be discussed by the TCT are immediately submitted into the \emph{Flagged} state.
-This is mainly used to discuss changes to the baseline documents and is also used to give formal DM approval to changes that DM are requesting to be made in project level documentation.
-When deemed necessary some RFCs may require a video meeting to allow more direct discussion.
+This is mainly used to discuss changes to the baseline documents and test reports associated with formal milestones, and is also used to give formal DM approval to changes that DM are requesting to be made in project level documentation.
```
Comment (ktlim) [inline rfc.tex]: Formally, no comma is used here.  You could rephrase to avoid the multiple "and"s.

File: rfc.tex [hunk 4]
```diff
@@ -40,5 +41,5 @@ \subsection{Change Control Board}
 Inside DM we have our own formal CCB that has evolved over the years from the SAT to something we called the Technical Control Team (TCT) but which has now been reorganized as the DM CCB.
 The DM CCB is chaired by the DM Systems Engineer and has representation from senior members of each DM team as well as the DM Project Manager and DM Subsystem Scientist.
 DM CCB uses the same RFC process as described above, but issues to be discussed by the TCT are immediately submitted into the \emph{Flagged} state.
```
Comment (ktlim) [inline rfc.tex]: Replace TCT here.

File: rfc.tex (hunk 2 above)
Comment (timj) [reply rfc.tex:15]: I really thought that we preferred RFCs where the proposer was going to take responsibility for implementation.

**Pull request merged**